setup.py

```diff
@@ -0,0 +1,16 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+A basic top-level setup.py script that delegates to the real one in
```

Comment:
Why is this necessary?
(our top-level directory gets crowded already)

Reply:
Installing a Python package from source requires a `setup.py` at the top-level of the archive, unfortunately.